tools/build/src/mocha-reporter/index.ts

```diff
@@ -69,30 +69,21 @@ class BentleyMochaReporter extends Spec {
   constructor(_runner: any, _options: any) {
     super(...arguments);
     this._junitReporter = new MochaJUnitReporter(...arguments);
-  }
-
-  public epilogue(...args: any[]) {
-    // Force test errors to be printed to stderr instead of stdout.
-    // This will allow rush to correctly summarize test failure when running rush test.
-    if (this.stats.failures) {
-      withStdErr(() => super.epilogue(...args));
-    } else {
-      super.epilogue(...args);
-
-      if (0 === this.stats.passes) {
-        logBuildError("There were 0 passing tests.  That doesn't seem right."
-          + "\nIf there are really no passing tests and no failures, then what was even the point?"
-          + "\nIt seems likely that tests were skipped by it.only, it.skip, or grep filters, so I'm going to fail now.");
-        failBuild();
-      }
-    }
 
     // Detect hangs caused by tests that leave timers/other handles open - not possible in electron frontends.
     if (!("electron" in process.versions)) {
+      process.on("chrome-test-runner-done", () => {
+        console.warn("WARNING: chrome-test-runner-done event received");
```

Comment:
Why is this a warning?  Won't this just add noise to our test output?

Reply:
That is a mistake (committing debug code). Will fix.

Reply:
Fixed.